pulp_python/app/serializers.py

```diff
@@ -22,6 +23,50 @@ class Meta:
         fields = ('name',)
 
 
+class DistributionDigestSerializer(serializers.ModelSerializer):
```

Comment:
Same here, we should validate that the digest type is supported and that the digest is the correct length for whatever type it is.

Reply:
I'm not sure about this, we really "support" any digest type. During sync time all we do is check that the type and value in the JSON API's digest field is the same.

Reply:
Good point, never mind then.